client/packages/invoices/src/Prescriptions/LineEditView/LineEditView.tsx

```diff
@@ -1,31 +1,54 @@
-import React, { useEffect, useMemo } from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import {
   BasicSpinner,
-  ModalMode,
+  InvoiceNodeStatus,
+  isEqual,
   NothingHere,
   RouteBuilder,
   useBreadcrumbs,
+  useConfirmOnLeaving,
+  useDirtyCheck,
+  useNavigate,
   useParams,
 } from '@openmsupply-client/common';
 
 import { ItemRowFragment, ListItems } from '@openmsupply-client/system';
 import { AppRoute } from '@openmsupply-client/config';
 import { PageLayout } from './PageLayout';
-import { usePrescription } from '../api';
+import { usePrescription, usePrescriptionLines } from '../api';
 import { AppBarButtons } from './AppBarButtons';
 import { PrescriptionLineEdit } from './PrescriptionLineEdit';
+import { DraftStockOutLine } from '../../types';
+import { Footer } from './Footer';
+import { NavBar } from './NavBar';
 
 export const PrescriptionLineEditView = () => {
   const { invoiceNumber, itemId } = useParams();
   const { setCustomBreadcrumbs } = useBreadcrumbs();
+  const navigate = useNavigate();
 
   const {
     query: { data, loading: isLoading },
   } = usePrescription();
 
+  const {
+    save: { saveLines, isSavingLines },
+    // delete: { deleteLines },
+  } = usePrescriptionLines(data?.id);
+
+  const newItemId = useRef<string>();
+
+  const { isDirty, setIsDirty } = useDirtyCheck();
+
   const lines =
-    data?.lines.nodes.sort((a, b) => a.item.name.localeCompare(b.item.name)) ??
-    [];
+    data?.lines.nodes.sort((a, b) => a.id.localeCompare(b.id)) ?? [];
```

Comment:
We need the order to be in the order that items were added (so the new item gets added to the bottom of the list). Fortunately, with v7 UUIDs, sorting by ID achieves this, as they have a built-in timestamp. I know relying on the ID for timestamp is not 100% reliable, but there's currently no other indicator of the actual added order of items to an invoice (is there?)

Reply:
I think this is ok, although I did kind of like the ordering by name... For a new item I agree it makes sense what you've done.

Reply:
The default ordering for the Detail view is in order added, so makes sense to be the same in here.